src/main/webapp/index.html

```diff
@@ -125,6 +125,7 @@
           <option value="definitions/alfresco-auth.yaml">Authentication API</option>
           <option value="definitions/alfresco-discovery.yaml">Discovery API</option>
           <option value="definitions/alfresco-workflow.yaml">Workflow API</option>
+          <option value="definitions/alfresco-sync-service.yaml">Sync Service API</option>
```

Comment:
I assume this API is only intended for Desktop Sync client. Since Sync Service API is *private* (as per url) it should not appear in the drop-down here (cc @ciprianchelaru) ?

Reply:
Hi Jan! For the moment this API is for Desktop Sync and Mobile team. After we made the merge we realized that we should change the API to be public and we already had created an issue for that https://issues.alfresco.com/jira/browse/DESKTOPAPP-31 which we'll be tackling in the next release train. To not complicate the situation, because the change is already merged, I think we can go at it is and in the next release train we will correct with the public API

Reply:
We need to remove it ... we can't add and then take away, it just confuses people especially since we dont know when API Explorer will be released & deployed again.

Reply:
we have to do revert for  PR #80 and PR #81

Reply:
@SimonaCarp made revert for those PR